src/test/ui/consts/miri_unleashed/ptr_arith.rs

```diff
@@ -14,7 +14,7 @@ static CMP: () = {
 static PTR_INT_CAST: () = {
     let x = &0 as *const _ as usize;
     //~^ ERROR could not evaluate static initializer
-    //~| unable to turn pointer into raw bytes
+    //~| "exposing pointers" needs an rfc before being allowed inside constants
```

Comment:
😆 well... that's not quite true, but considering it is very hard to reach without unleash...

Reply:
Should be impossible without unleash?

Also, we say the same thing about "pointer arithmetic or comparison"...^^